windows/src/engine/keyman32/Keyman32.cpp

```diff
@@ -679,9 +679,9 @@ BOOL UpdateRefreshTag(LONG tag)   // I1835 - Reduce chatter
 {
   PKEYMAN64THREADDATA _td = ThreadGlobals();
   if(!_td) return FALSE;
-  if(_td->RefreshTag_Process < tag)
+  if(_td->RefreshTag_Thread != tag)
```

Comment:
When Keyman is restarted, the global `RefreshTag` is reset to `0`, so we need to do a `!=` comparison rather than a `<` comparison to ensure that apps with an existing keyman32.dll/keyman64.dll in memory get refresh notifications (as their local `RefreshTag_Process` will likely already be greater than `0`).